include/tipsy/protocol.h

```diff
@@ -1,17 +1,18 @@
+#pragma once
+#ifndef TIPSY_ENCODER_PROTOCOL_H
+#define TIPSY_ENCODER_PROTOCOL_H
 /*
  * Given an encoder for raw binary, we need a communications protocol. This sort of
  * lays one out with a stateful encoder and decoder object.
  *
- * We have to take great care to have these be audio thread safe, so allocation and copies
+ * We have to take great care to have these be audio thread safe, so heap heap allocation and copies
```

Comment:
Heap heap? :)

Reply:
Ill fix that, thanks - it's a leftover from a bad merge resolution that I half/aborted